packages/toolkit/src/lib/react-query-service/organization/use-organizations/client.ts

```diff
@@ -0,0 +1,35 @@
+"use client";
+
+import type { ListOrganizationsRequest, Nullable } from "instill-sdk";
+import { useQuery } from "@tanstack/react-query";
+
+import { fetchOrganizations, getUseOrganizationsQueryKey } from "./server";
+
+export function useOrganizations({
+  accessToken,
+  payload,
+  enabled,
+}: {
+  accessToken: Nullable<string>;
+  payload: ListOrganizationsRequest;
+  enabled: boolean;
+}) {
+  let enabledQuery = false;
+
+  if (accessToken && enabled) {
+    enabledQuery = true;
+  }
+
+  const queryKey = getUseOrganizationsQueryKey();
+
+  return useQuery({
```

Comment:
Does it return the orgs based on the access token?

Reply:
yes, under the hood BE will use the accessToken to parse the correct user_id